modules/es-extensions/publisher/extensions/app/greg_publisher/themes/default/pages/add-list-password.hbs

```diff
@@ -0,0 +1,194 @@
+
+                                <div class="form-group">
+                                    <label class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12">Password <span class="required">*</span></label>
+                                    <div class="custom-form-right col-lg-5 col-md-8 col-sm-8 col-xs-12">
+                                        <input type="password" class="form-control" name="value" id="value" value="" required />
+                                    </div>
+                                </div>
+
+                                <div class="form-group">
+                                    <label class="custom-form-label col-lg-2 col-md-2 col-sm-12 col-xs-12">Re-enter Password <span class="required">*</span></label>
+                                    <div class="custom-form-right col-lg-5 col-md-8 col-sm-8 col-xs-12">
+                                        <input type="password" class="form-control" name="valueVerify" id="valueVerify" value="" required />
+                                    </div>
+                                </div>
+
+                                <div id="saveButtonsURL" class="form-group">
+                                    <div class="col-sm-offset-2 col-sm-10">
+                                        <input type="submit" value="Submit" class="btn btn-primary" />
+                                    </div>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <br>
+            <div>
+                {{#with  listassets.[0].context}}
+                    {{validate}}
+                {{/with}}
+            </div>
+
+            <div class="row-fluid">
+                <table class="table table-striped">
+                    <tr>
+                        <th>Server Name</th>
+                        <th></th>
+                    </tr>
+                    {{#with  listassets.[0].context}}
+                        {{#each passwords}}
+                            <tr>
+                                <td>
+                                    <label id="{{this.key}}" >{{this.key}}</label>
+                                </td>
+                                <td>
+                                    <a href="javascript:populate('{{this.key}}')" class="cu-btn-inner">
+                                        <span class="fw-stack">
+                                            <i class="fw fw-ring fw-stack-2x"></i>
+                                            <i class="fw fw-edit fw-stack-1x"></i>
+                                        </span> Edit
+                                    </a>
+                                </td>
+                            </tr>
+                        {{/each}}
+                    {{/with}}
+                </table>
+            </div>
+        </div>
+    </div>
+</div>
+
+<footer class="footer">
```

Comment:
Footer is removed and, added this hbs as a partial. So the default footer will be used.
PR #348